### ISSUE TITLE
Using momentary switches door is now only opening 1" and stopping. 

### ISSUE THREAD

**Issue by TTVert** [brbeaird/SmartThings_MyQ#117]
Close switch works fine.  When looking at MyQ app after using the momentary switch it just says "stopped" for that door.   This occurs for all 3 of my doors.  If i recreate the IFTTT applet it'll sometimes go up 3/4 of the way. The actual switches in smart things (not the momentary) are a bit confused also.  If i recall they all used to say "unknown" when working but now one door says unknown, one says opening and one say closing.  None of the 3 doors is functioning as designed this way though. They all still work in the standard MyQ app.  No idea what is going on.

**Comment by brbeaird** [comment]
I have not tried the momentary switches in awhile, but behind the scenes, all they do is call the same "open" command to MyQ as the other actions. I do know the no-sensor device type is still wonky in the new app, so you probably can't rely on that. However, you mentioned IFTTT - are you using that setup and working with virtual contact sensors? Wondering if that has anything to do with it.

Do you see any revealing messages in the IDE live logging when this happens?

**Comment by TTVert** [comment]
I am not using virtual contact sensors.  It's been a while since I set this up but I have IFTTT calling those momentary switches that I think were auto generated when I set it up.  It always had worked fine even though when I would press any of the momentary switches they'd say "Cannot connect to device.  Check device and try again".   I don't even know what i'm looking at here.

` Welcome back, db@cfbinc.com
Logout
My Locations
My Hubs
My Devices
My SmartApps
My Device Handlers
Live Logging
Documentation
All
garage door 1
MyQ Lite
garage door 2
Logs
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:23 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:23 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:23 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:22 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:22 PM: debug Opening!
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:22 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:22 PM: debug Opening!
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:22 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:22 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:21 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:21 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:21 PM: debug Opening!
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:21 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:21 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:21 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:20 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:19 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:19 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:20 PM: debug Opening!
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:19 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:19 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:19 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:10:19 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 35640fb8-e93d-4558-a40e-44315f67f59e DNI: CG08301580F7 Opener
d24f2c8b-c0bc-48ca-bfba-ddab0d39d2fc 12:10:19 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:57 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:57 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:57 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:56 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:56 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:55 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:55 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:55 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:55 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:55 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug got accountid 248d0abc-117a-4006-a985-4c88fa427fc4
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got result: 0
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 200
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got LOGIN POST response: STATUS: 200

DATA: [SecurityToken:dec93763-29f6-408e-8371-175b2917aafa]
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: warn Token has expired. Logging in again.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 204
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug got accountid 248d0abc-117a-4006-a985-4c88fa427fc4
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got result: 0
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:54 PM: debug Got response: STATUS: 200
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:53 PM: debug Got LOGIN POST response: STATUS: 200

DATA: [SecurityToken:b1b4a598-0b44-4d1e-a708-5479c8598d16]
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:53 PM: warn Token has expired. Logging in again.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:53 PM: debug Opening door.
af667657-0af1-4acc-be9d-3b89a7700c57 12:09:53 PM: debug Door open button push detected: Event name momentary value: pushed deviceID: 4258cc02-fdc6-49a4-93e5-4860eb2948e1 DNI: CG083013E733 Opener
cb801bc6-82c4-4545-9623-f495840af94f 12:09:54 PM: debug Opening!
cb801bc6-82c4-4545-9623-f495840af94f 12:09:53 PM: debug Opening!
 12:08:22 PM: info Waiting on events...
 12:08:22 PM: info For past logs for individual things go to the My Devices section, find the device and click on the Events link on the device information page.
 12:08:22 PM: info This console provides live logging of your SmartThings.`

**Comment by brbeaird** [comment]
Weird. Almost looks like it's sending two requests at once - maybe that's causing the door to pause?

Maybe you could try removing the door and momentary switches and then re-adding. I know that's a lot of trouble, but it's probably the best thing to try.

**Comment by TTVert** [comment]
I did remove the IFTTT stuff.  I know i can remove the momentary switches via the smart things api but how do I go about re-adding them?  It's been quite some time now.  I do appreciate you helping me out w/ this.

**Comment by brbeaird** [comment]
If you go into the MyQ SmartApp setup, you can uncheck the box to create momentary buttons. Tap through the process and it should delete those devices. I'm pretty sure you could also just delete them from the mobile app directly or through the IDE.

**Comment by TTVert** [comment]
I'm afraid this may have been your typical user error.  I checked my Device handler data for the Virtual Switch Tile and it somehow was not updated when I updated the smart app and the MyQ Garage Door Opener-NoSensor device handler.  The virtual switches actaully work onw also whereas before they didn't when you pressed on them

The one thing I'm running into is w/ IFTTT.  Before the image below would show all the momentary buttons created but it is only showing the one virtual switch I created.  Have any input on that?  I guess in theory I can create 2 virtual switches for each of my doors but i'd rather not (6 total switches)
![image](https://user-images.githubusercontent.com/24374538/106816241-b70ab280-663a-11eb-92bf-7934e3a47838.png)
